pipeline.yaml

```diff
@@ -5,6 +5,11 @@ metadata:
 spec:
   limits: 
     readBatchSize: 1
+  watermark:
```

Comment:
Did we see getPartitions being invoked with this spec? If so, we can remove this spec from the example. It served its purpose of testing and it might not be necessary to have idling feature in our examples.